src/discordcr/rest.cr

```diff
@@ -404,6 +404,10 @@ module Discord
         end
       end
 
+      if spoiler
```

Comment:
Maybe we should avoid prepending this again if it is already provided by the user or `File`.
```suggestion
      if spoiler && !filename.starts_with?("SPOILER_")
```